tasks/ceph_deploy.py

```diff
@@ -393,6 +398,50 @@ def execute_ceph_deploy(cmd):
                               'grep', '-v', 'grep', run.Raw('|'),
                               'grep', 'ceph'], check_status=False)
 
+	log.info('Checking cluster log for badness...')
+        def first_in_ceph_log(remote, pattern, excludes):
```

Comment:
This indentation looks weird. Why define this function in the middle of another function?